runners/maven-plugin/src/main/kotlin/DokkaMojo.kt

```diff
@@ -166,7 +166,7 @@ abstract class AbstractDokkaMojo : AbstractMojo() {
     var failOnWarning: Boolean = DokkaDefaults.failOnWarning
 
     @Parameter
-    var dokkaPlugins: List<Dependency> = emptyList()
+    open var dokkaPlugins: List<Dependency> = emptyList()
```

Comment:
Cannot we pass this list as constructor parameter for this abstract class?

Reply:
I just missed that option, will change